docs/user-guide/ui-access.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Nice! But this file needs a spot in the left nav. Can you add one in guides.yml?